ServiceBus/ServiceBus/EntityDataModel.edmx.sql

```diff
@@ -0,0 +1,135 @@
+
+-- --------------------------------------------------
+-- Entity Designer DDL Script for SQL Server 2005, 2008, 2012 and Azure
+-- --------------------------------------------------
+-- Date Created: 10/29/2016 18:40:01
+-- Generated from EDMX file: D:\Visual Studio workplace\ATNET2016\ServiceBus\ServiceBus\EntityDataModel.edmx
+-- --------------------------------------------------
+
+SET QUOTED_IDENTIFIER OFF;
+GO
+USE [D:\VISUAL STUDIO WORKPLACE\ATNET2016\SERVICEBUS\SERVICEBUS\APP_DATA\DATABASE.MDF];
+GO
+IF SCHEMA_ID(N'dbo') IS NULL EXECUTE(N'CREATE SCHEMA [dbo]');
+GO
+
+-- --------------------------------------------------
+-- Dropping existing FOREIGN KEY constraints
+-- --------------------------------------------------
+
+IF OBJECT_ID(N'[dbo].[FK_BasketCampaign_Basket]', 'F') IS NOT NULL
+    ALTER TABLE [dbo].[BasketCampaign] DROP CONSTRAINT [FK_BasketCampaign_Basket];
+GO
+IF OBJECT_ID(N'[dbo].[FK_BasketCampaign_Campaign]', 'F') IS NOT NULL
+    ALTER TABLE [dbo].[BasketCampaign] DROP CONSTRAINT [FK_BasketCampaign_Campaign];
+GO
+
+-- --------------------------------------------------
+-- Dropping existing tables
+-- --------------------------------------------------
+
+IF OBJECT_ID(N'[dbo].[Products]', 'U') IS NOT NULL
+    DROP TABLE [dbo].[Products];
+GO
+IF OBJECT_ID(N'[dbo].[Basket]', 'U') IS NOT NULL
+    DROP TABLE [dbo].[Basket];
+GO
+IF OBJECT_ID(N'[dbo].[Campaign]', 'U') IS NOT NULL
+    DROP TABLE [dbo].[Campaign];
+GO
+IF OBJECT_ID(N'[dbo].[BasketCampaign]', 'U') IS NOT NULL
+    DROP TABLE [dbo].[BasketCampaign];
+GO
+
+-- --------------------------------------------------
+-- Creating all tables
+-- --------------------------------------------------
+
+-- Creating table 'Products'
+CREATE TABLE [dbo].[Products] (
+    [Id] uniqueidentifier  NOT NULL,
+    [Name] nvarchar(50)  NOT NULL,
+    [Price] float  NOT NULL
+);
+GO
+
+-- Creating table 'Basket'
+CREATE TABLE [dbo].[Basket] (
+    [Id] uniqueidentifier  NOT NULL
+);
+GO
+
+-- Creating table 'Campaign'
+CREATE TABLE [dbo].[Campaign] (
+    [Id] uniqueidentifier  NOT NULL,
+    [Name] nvarchar(max)  NOT NULL,
+    [Discount] float  NOT NULL
+);
+GO
+
+-- Creating table 'BasketCampaign'
+CREATE TABLE [dbo].[BasketCampaign] (
+    [BasketCampaign_Campaign_Id] uniqueidentifier  NOT NULL,
+    [Campaigns_Id] uniqueidentifier  NOT NULL
+);
```

Comment:
I guess, there should be also something like... [Basket_id] uniqueidentifier NOT NULL
So that each basket campaign is connected to a specific basket